docs/integrations/wallets.md

```diff
@@ -0,0 +1,108 @@
+# Wallets
+
+If you are an Ethereum wallet provider, you can integrate Lido staking into your app or website and earn [Lido referral rewards](https://lido.fi/referral). 
+
+## Your referral link
+
+You can start referring people simply by sharing your special referral link to Lido [Staking Widget](https://stake.lido.fi/). Your referral link looks like this `https://stake.lido.fi/?ref=YOUR_REWARDS_ADDRESS` where `YOUR_REWARDS_ADDRESS` is your Ethereum address or ENS name registered for the Lido Referral Program. Anyone who uses your link to stake will automatically set your rewards address as their referral.
+
+For example, Vitalik's referral link is [`https://stake.lido.fi/?ref=0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B`](https://stake.lido.fi/?ref=0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B). 
```

Comment:
Is Vitalik whitelisted? :exploding_head:

Reply:
I don't think so. I needed some real-world address :)

Reply:
Changed to "would be" to avoid confusion.

Reply:
thx